projects/ngx-grapesjs/schematics/ng-add/index.ts

```diff
@@ -13,18 +13,13 @@ export const ngAdd =
     const grapesJsAssetPaths = [`${baseGrapesjsPath}/grapes.min.js`];
 
     if (options.editorType === 'webpageEditor') {
-      grapesCssAssetPaths.push(
-        `${baseGrapesjsWebpagePath}/grapesjs-preset-webpage.min.css`
-      );
       grapesJsAssetPaths.push(
-        `${baseGrapesjsWebpagePath}/grapesjs-preset-webpage.min.js`
+        `${baseGrapesjsWebpagePath}/index.js`,
+        'node_modules/grapesjs-blocks-basic/dist/index.js'
       );
     } else {
-      grapesCssAssetPaths.push(
-        `${baseGrapesjsNewsletterPath}/grapesjs-preset-newsletter.css`
-      );
       grapesJsAssetPaths.push(
-        `${baseGrapesjsNewsletterPath}/grapesjs-preset-newsletter.min.js`
+        `${baseGrapesjsNewsletterPath}/index.js`
       );
     }
     return chain([
```

Comment:
Since the `grapesCssAssetPaths` is used only once, we can set its value directly to the `addGrapesJsCssToTarget` method.